data/en/bots/null.md

```diff
@@ -0,0 +1,14 @@
+---
+avatar: null
+cover: null
+custom_path: null
+description: This bot allows you to do magic
```

Comment:
Common dude.... Put at least SOME effort into that.

Reply:
sorry, dont know how i got here, did this by accident.

Reply:
ignore this post